support-frontend/package.json

```diff
@@ -193,5 +193,6 @@
 	},
 	"lint-staged": {
 		"./assets/**/**/*.{js,jsx,ts,tsx}": "eslint"
-	}
+	},
+	"packageManager": "yarn@1.22.22"
```

Comment:
Presume this what  `core_pack enable` is using to update and maybe chnaged the pnpm in future?